README.md

```diff
@@ -27,12 +27,114 @@ The documentation of the 121 platform can be found on the Wiki of this repositor
+### Install Git
+
+        sudo apt install git-all
+
+### Install Docker
+
+On linux distributions we need to install docker enginer and docker compose respectively. On other platforms docker-compose enginer is available through Docker Desktop. Read more at: https://docs.docker.com/engine/install/
+
+Docker compose relies on docker engine, so in order to get started we must install docker-engine first. (Read more at: https://docs.docker.com/engine/install/ubuntu/). Theere are various ways of installing docker engine (repository, packages or automated scripts). The official recommended way doing that is through setting up a repository.
+
+#### First off, uninstall any old version of docker-engine.
+
+    sudo apt-get remove docker docker-engine docker.io containerd runc
+
+#### Choose an installation method from the provided list at https://docs.docker.com/engine/install/ubuntu/#installation-methods and install docker-engine
+
+#### Test your docker installation
+
+     sudo docker run hello-world
+
+#### Now install docker-compose
+Can be done by following the steps at: https://docs.docker.com/compose/install/
+
+    sudo curl -L "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+
+    sudo chmod +x /usr/local/bin/docker-compose
+
+#### Test the docker compose installation
+
+    docker-compose --version
+
+You are set!
+
+# 3. MacOS
+## Install dependencies
+### Install Node.js (https://nodejs.org/en/download/)
+#### Choose an appropriate executable to install node.js
+### Install Git https://git-scm.com/book/en/v2/Getting-Started-Installing-Git > Installing on macOS (section)
+### Install Docker (https://docs.docker.com/docker-for-mac/install/)
+
+
+Once the dependencies are resolved depending on your OS, as above we can set-up the repositories and other environment variables
+
+## Setup Repository
+Download/Clone the Git repository.
+
+    git clone https://github.com/global-121/121-platform.git
+
+
+## Setup Interfaces
+Install dependencies for interfaces. Through command line navigate to each interface directory and install dependencies
+
+    cd interfaces/AW-App
+    npm install
+
+    cd interfaces/HO-Portal
+    npm install
+
+    cd interfaces/PA-App
+    npm install
+
+## Setup services
+
+Switch to the repository folder
+
+    cd services/
+
+Copy a few secret files and get the right passwords:
+
+    cp .env.example .env
+    cp 121-service/src/example.secrets.ts 121-service/src/secrets.ts
+    cp 121-service/example.ormconfig.json 121-service/ormconfig.json
+    cp PA-accounts-service/src/example.secrets.ts PA-accounts-service/src/secrets.ts
+    cp PA-accounts-service/example.ormconfig.json PA-accounts-service/ormconfig.json
+
+Environment variables are explained in the comments of the .env.example
+
+
+You can start in the development mode by following:
 - Run: `npm run install:all` from *this* folder
 - Run: `npm run start:all` from *this* folder
```

Comment:
Not clear what "this" refers to. I assume it means: run the commands from the services folder.

Reply:
Adding the related clarification.